src/libs/Network.js

```diff
@@ -256,6 +256,24 @@ function canProcessRequestImmediately(request) {
     return lodashGet(request, 'data.shouldProcessImmediately', true);
 }
 
+/**
+ * @param {Object} data
+ * @returns {Object}
+ */
+function normalizeData(data) {
```

Comment:
I can remove this if needed. Just figured that we will get bit by this again and can't think of any places where we are calling API with an array parameter. But if we are it's an array of strings or numbers and `Get` should be able to handle it...

Gonna do a quick sweep of the different usages.

Reply:
Actually maybe it is safer to not do this but do a warning instead in case fixing something breaks other stuff 🤷‍♂️

Reply:
Agreed, since we're CPing this I think we should just fix the policyID case and create a separate GH for fixing formData for all Network calls

Reply:
yea gonna do it in a follow up and run through regular regressions.